Ask user for old service file

Pitillo opened the pull request:
As discussed [here](https://github.com/evcc-io/evcc/discussions/2923) I've added questions to the postinst file to ask the user whether he wants to keep the service file under /etc/systemd/system/evcc.service and the binary under /usr/local/bin/evcc

For now I didn't have time to test this. If anyone can do so, feel free.

Commit:
Added questions for service and alternate binary on installation

andig commented:
Sieht Klasse aus, bräuchte aber noch den erwähnten Test. Kann ich unter OSX leider nicht machen :/

Commit:
bugs fixed:

- shell doesnt know source, only .
- && instead of -a
- copy paste errors
- set -e doesnt allow for examining the exitstatus, has to be done in same line using ||

Pitillo commented:
Ich habe mir jetzt heute die Zeit genommen und das ganze Mal getestet. Ein paar kleine Bugs waren drin. Hier der Output des Tests:
```root@JHPC:~/projects/evcc# cp /lib/systemd/system/evcc.service /etc/systemd/system/evcc.service
root@JHPC:~/projects/evcc# rm /etc/evcc-userchoices.sh
root@JHPC:~/projects/evcc# dpkg -i release/evcc_0.89.0_amd64.deb
(Reading database ... 41569 files and directories currently installed.)
Preparing to unpack release/evcc_0.89.0_amd64.deb ...
Unpacking evcc (0.89.0) over (0.89.0) ...
Setting up evcc (0.89.0) ...
An alternate service file was detected under '/etc/systemd/system/evcc.service'.
This is probably due to a previous manual installation.
You probably want to delete this file now. Your evcc configuration stays untouched!
Shall '/etc/systemd/system/evcc.service' be deleted? [Y/n]:
y
The file will be deleted.
Deleting old service file '/etc/systemd/system/evcc.service'
removed '/etc/systemd/system/evcc.service'
root@JHPC:~/projects/evcc# ls /etc/systemd/system/evcc.service
ls: cannot access '/etc/systemd/system/evcc.service': No such file or directory
root@JHPC:~/projects/evcc# cp /lib/systemd/system/evcc.service /etc/systemd/system/evcc.service
root@JHPC:~/projects/evcc# dpkg -i release/evcc_0.89.0_amd64.deb
(Reading database ... 41569 files and directories currently installed.)
Preparing to unpack release/evcc_0.89.0_amd64.deb ...
Unpacking evcc (0.89.0) over (0.89.0) ...
Setting up evcc (0.89.0) ...
An alternate service file was detected under '/etc/systemd/system/evcc.service'.
This is probably due to a previous manual installation.
You probably want to delete this file now. Your evcc configuration stays untouched!
Shall '/etc/systemd/system/evcc.service' be deleted? [Y/n]:
n
Ok. We will keep that file. Keep in mind that you may need to alter it if any changes are done upstream. Your answer is saved for the future.
root@JHPC:~/projects/evcc# ls /etc/systemd/system/evcc.service
-rw-r--r-- 1 root root 304 Apr  3 19:32 /etc/systemd/system/evcc.service
root@JHPC:~/projects/evcc# dpkg -i release/evcc_0.89.0_amd64.deb
(Reading database ... 41569 files and directories currently installed.)
Preparing to unpack release/evcc_0.89.0_amd64.deb ...
Unpacking evcc (0.89.0) over (0.89.0) ...
Setting up evcc (0.89.0) ...
root@JHPC:~/projects/evcc#```

andig commented:
Klasse- also rein damit?

Pitillo commented:
Aus meiner Sicht schon, also habe es halt wie im Log zu sehen rudimentär mit 3 mal installieren ausprobiert und es scheint genau so zu funktionieren, wie ich es mir vorstellen würde :D

andig commented:
Top, vielen Dank!
